client/src/test/java/TestRunner.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
```

Comment:
The bash script should be fine for linux and darwin, but could you also include a windows batch script?

Reply:
Added bat Script for Windows in the latest commit. f697089707e1581fa057312389b98cd66fd85458